x-pack/legacy/plugins/apm/public/utils/formatters.ts

```diff
@@ -115,23 +115,26 @@ export type TimeFormatter = (
 
 type TimeFormatterBuilder = (max: number) => TimeFormatter;
 
-export const getTimeFormatter: TimeFormatterBuilder = memoize((max: number) => {
-  const unit = timeUnit(max);
-  switch (unit) {
-    case 'h':
-      return asHours;
-    case 'm':
-      return asMinutes;
-    case 's':
-      return asSeconds;
-    case 'ms':
-      return asMillis;
-    case 'us':
-      return asMicros;
+export const getDurationFormatted: TimeFormatterBuilder = memoize(
```

Comment:
`getDurationFormatted` implies that the function returns a formatted duration. It doesn't. Instead it returns a formatter function that can return a formatted duration. Therefore I think it's better to call it `getDurationFormatter`